udata/tests/frontend/test_markdown.py

```diff
@@ -108,7 +108,7 @@ def test_markdown_linkify_https(self, md2dom):
         assert el.getAttribute('data-tooltip') == ''
         assert el.firstChild.data == 'foo'
 
-    def tesft_markdown_linkify_ftp(self, md2dom):
+    def test_markdown_linkify_ftp(self, md2dom):
```

Comment:
this typo was caused by doing `cmd+maj+f` at the time but my macbook air keyboard is sometimes ignoring some letters (or typing them several times)

I fixed the typo and I had rerun the test before pushing